.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
the build with 15.x was [failing](https://github.com/lquixada/cross-fetch/runs/4887287308?check_suite_focus=true) for a seemingly unrelated reason.

According to the matrix, 15 is no longer supported, so I wanted to try to see if it will work with 16.x

Reply:
makes sense to me!